sycl/test-e2e/WeakObject/weak_object_copy.cpp

```diff
@@ -0,0 +1,30 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
```

Comment:
Do we need to run these tests on each device? It doesn't seem like we actually submit any kernels, so this seems excessive.

Reply:
Do we have a general placeholder? I worry that if we only run it for one of these, we could have systems that silently just ignore the tests because they don't have the device we picked.

Reply:
`%BE_RUN_PLACEHOLDER` should do the trick.

Reply:
Good call! That is now being used instead.